CONTRIBUTING.md

```diff
@@ -89,6 +89,42 @@ After build completes, you can preview the resulting site locally using built-in
 	
 This will start the local preview on localhost:8080. You can then view the changes by going to http://localhost:8080/docs/[path] (e.g. http://localhost:8080/docs/getting-started/).   
 
-## Contributing to samples
+# Contributing to samples
 
-See the [Samples Readme](https://github.com/dotnet/core-docs/blob/master/samples/README.md).
+We encourage samples for any of our topics. Readers can download and run the samples. They
+can explore the code and experiment with their own changes.
+
+All samples should be complete applications or libraries. Where the sample creates a library,
+it should include unit tests or an application that lets readers run the code.
+
+The samples are organized in directories that match the directories under
```

Comment:
@richlander, a couple of comments on this: 
> I'd love to see us link to the Core contributing docs, since they cover some important and complementary content: https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/contributing.md.

I think that adding another link here with a lot of things that don't apply to us is confusing. We already link to other materials that they need to read. We should keep this simple IMO. So whatever is important that we're missing  should be added here no?

> Each author should have a link to an online presence for himself/herself.

Not applicable anymore

> Step 1: Open an Issue describing the article you wish to write and how it relates to existing content. Get approval to write your article.

Agree. Fixed. 

> The documentation is written in GitHub-flavored Markdown and built using docfx. The documented is hosted at docs.microsoft.com.

Changed.

> we should include a statement on when docfx will move off of DNX to .NET Core 1.0

I'd have to check when this is actually scheduled to happen. 

> The conceptual documentation is placed in the docs folder in the root of the repo so most of the work will happen there.

Not applicable anymore.

> Does "path" take you to a specific location or a 404?

No. I've previously added an example that actually takes you to an actual location. See if that would work for you.